ui/webui/src/components/AnacondaAboutModal.jsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from "react";
+import {
+    AboutModal,
+    Button,
+    Flex,
+    TextContent,
+    TextList,
+    TextListItem,
+} from "@patternfly/react-core";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+import { getAnacondaVersion } from "../helpers/product";
+import "../components/app.scss";
+
+export const AnacondaAboutModal = ({ isModalOpen, setIsAboutModalOpen }) => {
+    const toggleModal = () => {
+        setIsAboutModalOpen(!isModalOpen);
+    };
+
+    const [anacondaVersion, setAnacondaVersion] = useState("");
+
+    useEffect(() => {
+        getAnacondaVersion().then(content =>
+            setAnacondaVersion(content.split(" ").slice(-1)[0].replace("\n", ""))
+        );
+    }, []);
+
+    return (
+        <>
+            <AboutModal
+              style={{ marginTop: "0" }}
+              isOpen={isModalOpen}
+              onClose={toggleModal}
+              trademark="© 2023 Red Hat, Inc."
```

Comment:
This might need to be adjusted to say "Anaconda © 2023 Red Hat, Inc.", else it might look like RH is trying to claim copyright on the distribution (including Alma and Rocky) and all packages listed on the screen.